model/src/main/java/uk/gov/pay/commons/model/SupportedLanguageJsonDeserialize.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.pay.commons.model;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+
+import java.io.IOException;
+
+public final class SupportedLanguageJsonDeserialize extends JsonDeserializer<SupportedLanguage> {
+    @Override
+    public SupportedLanguage deserialize(final JsonParser parser, final DeserializationContext context) throws IOException {
+        final String valueAsString = parser.getValueAsString();
```

Comment:
total nitpick, is this worth the extra line?

Reply:
Well, it depends how readable you want your code to be.